platform/stm32f0xx/platform.mk

```diff
@@ -22,7 +22,7 @@ LDSCRIPT_DIR := $(PLATFORM_DIR)/ldscripts
 SCRIPT_DIR := $(PLATFORM_DIR)/scripts
 
 # Build flags for the device
-CDEFINES := USE_STDPERIPH_DRIVER STM32F072 SEMIHOSTING_ENABLED
+CDEFINES := USE_STDPERIPH_DRIVER STM32F072 SEMIHOSTING_ENABLED=1
```

Comment:
This can be left as SEMIHOSTING_ENABLED. If not explicitly set, defines are truthy by default. :)